### ISSUE TITLE
Softmax as logistic function

### ISSUE THREAD

**Issue by maomran** [www0wwwjs1/Matrix-Capsules-EM-Tensorflow#22]
Hi, 
I have two questions: 
1) After training mnist, the testing it gets me average accurage of 0.3, any ideas why?
2) Activations are updated by calculating the cost and applying a logistic function, why is a softmax used instead of sigmoid?

**Comment by www0wwwjs1** [comment]
Hi,

1. Basically, you can use the hyper parameters given in the readme file for similar performance on mnist. Other parameters may lead to failure in training.
2. Sigmoid function can give a similar performance according to our experiments. Softmax here just theoretically ensures a distribution over capsules.
